pages/CONTRIBUTING-docs.md

```diff
@@ -0,0 +1,72 @@
+# JSON Schema documentation contributing guide 
+
+This is guide describes the workflow to contribute to the JSON Schema technical documentation. If you would like to contribute to other aspects of the JSON Schema website, like writing blog articles or publishing a case study, please read the website's [Contributing Guide](/website/CONTRIBUTING.md) for more details. 
```

Comment:
"To contribute to" and "for contributing to" are grammatically correct. I use "to contribute to" because I read it's the most commonly used. Apparently, "for contributing to" is often used when expressing gratitude "Thank you for contributing to the docs".